pdfarranger/pdfarranger.py

```diff
@@ -1252,7 +1254,7 @@ def save_warning_dialog(self, msg):
     def export_finished(self, exportmode, export_msg):
         """Check if export finished. Show any messages. Run any post action."""
         if self.export_process.is_alive():
-            return True
+            return True # continue polling
```

Comment:
"Inline comments should be separated by at least two spaces from the statement." :point_up:

Reply:
Good catch, thanks!